### ISSUE TITLE
Auto hide adding time when hovered.

### ISSUE THREAD

**Issue by AllovueCharly** [mansona/ember-cli-notifications#217]
When hovering over a notification, the timer at the bottom of the notification stops, but when you stop hovering and let the time run out, the notification stays for few more seconds. It seems that each additional hover adds more time to the auto hide at the end of the initial counter.

**Comment by mansona** [comment]
@AllovueCharly thanks for your bug report! Can you verify that this is still a problem? We have shipped a reasonably large update with https://github.com/mansona/ember-cli-notifications/pull/251 and it might have automatically fixed this issue

**Comment by AllovueCharly** [comment]
We are currently on 4.3.3, but when we bump up to 5.1.0, I will check to see if this is still a problem. Would it be okay if I closed this ticket once we update and I can verify this for you?

**Comment by mansona** [comment]
@AllovueCharly yes feel free to close this ticket at any time! You can even close it and re-open it later if that suits 😄
